### ISSUE TITLE
Setup

### ISSUE THREAD

**Issue by DominicFrei** [DominicFrei/iOS_Localised_System_Buttons#1]
@zkirill So, what are your targeting for?
Just plain localisation files or more?

**Comment by zkirill** [comment]
I use https://github.com/mobiata/twine but we should keep things as simple as possible so plain localization files work. Also, I'm not positive that no two (or more) native elements do not share the same accessibility label.

Something like this?

zh-Hans.lproj/Localizable.strings
```
/**
 * Apple Strings File
 * Language: zh-Hans
 */

/* UIBarButtonSystemItem.Cancel */
"Cancel" = "取消";
/* UISearchBar text field. */
"Search" = "搜索";
/* UISearchController table view. */
"Search results" = "搜索结果";
```

**Comment by DominicFrei** [comment]
Yes, that is a very valid point. We will find out eventually I guess ..

Yes, the structure looks good to me. If we need something else, we can change it later on.

On the other side, we could additionally provide something like https://github.com/mac-cain13/R.swift for this project.
I like the idea of strong typing instead of using strings. What do you think?

**Comment by DominicFrei** [comment]
https://github.com/DominicFrei/iOS_Localised_System_Buttons/blob/master/de.lproj/Localizable.strings

**Comment by DominicFrei** [comment]
Starting with this I will continue with the ```UIBarButtonSystemItem```s.

**Comment by zkirill** [comment]
Cool, haven't seen R before but I did come across https://github.com/JiriTrecak/Laurine a few days ago which seems to do something similar?

I also prefer strong typing and it makes sense in this case because the enduser of this library would not be familiar with the new string format and will have to look up the key for each string individually while writing tests. It would also help us if we, for some reason, need to change the key/value due to error or otherwise.

My only concern is asking the enduser to learn/implement R (or some alternative) which may add extra difficulty for non-English speakers so maybe we show instructions for both vanilla and R implementation in the README?

**Comment by DominicFrei** [comment]
Yeah, Laurine is something I found just yesterday. There are so many awesome frameworks out there!

Oh, I was not thinking about using R for this project. I thought about our own, maybe less complex system of strong typing. But to be honest, I never really used R.Swift. Just heard about it. Maybe it is really exactly the same for images as our idea. In that case, not doing the same things twice is or course a valuable idea.

**Comment by DominicFrei** [comment]
<img width="1388" alt="screen shot 2015-12-15 at 21 49 18" src="https://cloud.githubusercontent.com/assets/1942012/11823693/ba9ba36a-a375-11e5-9fe9-b1ff9e24b035.png">

Alright, the setup for the system buttons is done. I'll use the UI tests to make sure I will get exactly the name I need. :+1:

**Comment by DominicFrei** [comment]
I started with the German version:
https://github.com/DominicFrei/iOS_Localised_System_Buttons/blob/master/de.lproj/Localizable.strings

**Comment by zkirill** [comment]
Awesome!!! :stars: I will add a few `zh-Hans`, `zh-Hant`, and `ja` in a few hours. Thanks! :smiley:

**Comment by DominicFrei** [comment]
:+1:

**Comment by DominicFrei** [comment]
I created an English versioon, too.
Even though the keys are obvious, we need that for the later strong typing anyway:

https://github.com/DominicFrei/iOS_Localised_System_Buttons/blob/master/en.lproj/Localizable.strings
